src/test/java/com/marizueva/laboratory/hw5/steps/JdiSiteUserTablePageSteps.java

```diff
@@ -0,0 +1,108 @@
+package com.marizueva.laboratory.hw5.steps;
+
+import com.marizueva.laboratory.hw5.context.TestContext;
+import com.marizueva.laboratory.hw5.pages.UserTablePage;
+import com.marizueva.laboratory.hw5.utils.UsedInTestTerms;
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.cucumber.messages.Messages;
+import org.openqa.selenium.WebElement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class JdiSiteUserTablePageSteps {
+    UserTablePage userTablePage;
+
+    {
+        userTablePage = new UserTablePage(TestContext.getInstance()
+                .getTestObject(TestContext.WEB_DRIVER));
+    }
+
+    @When("I select 'vip' checkbox for 'Sergey Ivan'")
+    public void chooseVipCheckboxForSergeyIvan() {
+        userTablePage
+                .chooseVipCheckboxForSergeyIvan();
+    }
+
+    @Then("1 log row has 'Vip: condition changed to true' text in log section")
+    public void vipConditionChangedToTrueLog() {
+        List<String> actualallLogs = userTablePage
+                .getAllLogs();
+
+        assertEquals(actualallLogs.size(), 1);
+        assertEquals(actualallLogs.get(0).substring(9), UsedInTestTerms.userPageVipLog);
+
+    }
+
+    @Then("'User Table' page should be opened")
+    public void userTablePageOpened() {
+        boolean userTableOpened = userTablePage.isUserTableOpened();
+
+        assertTrue(userTableOpened);
+    }
+
+    @Then("6 Number Type Dropdowns should be displayed on Users Table on User Table Page")
```

Comment:
Amount of the dropdowns should be as a parameter

Reply:
fixed